tests/cython/CMakeLists.txt

```diff
@@ -22,6 +22,22 @@ IF (HAVE_SERVER)
 ENDIF (HAVE_SERVER)
 
 
+# The PythonEvalUTest is derived from the CogServer class, and so 
+# cannot be tested unless the cogserver is actually being built!
+IF (HAVE_SERVER)
```

Comment:
In this case, I wanted to create test case as close to bug #1191 as possible. Since that bug uses the Python Shell via the Cogserver, I figured following the lead of PythonModuleUTest was as close to the server path as I could get.